CLogger/src/Utils/Utils.cpp

```diff
@@ -0,0 +1,38 @@
+#include "stdafx.h"
+
+#include "Utils/Utils.h"
+
+namespace LogUtils
+{
+	std::string AddZero(const int value)
+	{
+		auto str = std::to_string(value);
+
+		if (value < 10)
+		{
+			str.insert(str.begin(), '0');
+		}
+
+		return str;
+	}
+	
+	std::string GetCurrentTimeString()
```

Comment:
I don't know, but maybe I have to make this util-function `inline`...

Reply:
Could be inline

Reply:
Good, so it will be done